src/ipv4.rs

```diff
@@ -23,48 +27,58 @@ pub struct IPv4Header {
     pub dest_addr: Ipv4Addr,
 }
 
+// To remove ?
```

Comment:
I think we should not make this function public, this is likely to be use only by us, in fact I would prefer inline this unwrap call.